src/cpu/core_dynrec/decoder_basic.h

```diff
@@ -286,9 +291,9 @@ static void INLINE decode_increase_wmapmask(Bitu size) {
 	}
 	// update mask entries
 	switch (size) {
-		case 1 : activecb->cache.wmapmask[mapidx]+=0x01; break;
-		case 2 : (*(Bit16u*)&activecb->cache.wmapmask[mapidx])+=0x0101; break;
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the host_* functions do).